css/styles.css

```diff
@@ -437,31 +568,22 @@ img {
   }
 
 .highlighting:hover::before,
-.highlighting:hover::after {
+.highlighting:hover::after,
+.highlighting:focus::before,
+.highlighting:focus::after,
+.highlighting:active::before,
+.highlighting:active::after {
   opacity: 1;
 }
 
-.about-us-number {
-  position: absolute;
-  top: 30px;
-  left: 30px;
-  z-index: -1;
-  font-family: "Bakbak One", Arial, sans-serif;
-  font-size: 80px;
-  line-height: 1.2;
-  text-align: left;
-  color: transparent;
-  -webkit-text-stroke-width: 1px;
-  -webkit-text-stroke-color: #21e786;
-}
-
 .about-us-item-title {
   font-size: 24px;
   line-height: 1.25;
   text-align: left;
```

Comment:
This property no needed here

Reply:
![about-us wrapper before - 2](https://github.com/user-attachments/assets/f6027d05-e67d-48fb-94fb-a83eadb45d0a)
![about-us wrapper before](https://github.com/user-attachments/assets/9df272d4-f7dc-41bc-94b8-8dfaf595e7aa)

I've tried to delete these property and it doesn't work without them. I mean there is no visibility of ::before element.

Reply:
I told about `text-align: left` only, no need to remove other properties )